lib/src/list_tile.dart

```diff
@@ -290,35 +286,30 @@ enum _ListTileSlot {
 
 class _ListTile extends RenderObjectWidget {
   const _ListTile({
-    Key key,
+    Key? key,
     this.separator,
     this.leading,
     this.title,
```

Comment:
doesn't make much difference, but i noticed that the material version in flutter has marked this as required: https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/material/list_tile.dart#L1209

probably because it cannot be null? https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/material/list_tile.dart#L1116

Reply:
Good catch. I've added it.